fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/AsyncWebServiceRequestHandler.java

```diff
@@ -32,8 +44,7 @@
      * @param mapper  The mapper for all JSON processing
      */
     public AsyncWebServiceRequestHandler(DruidWebService druidWebService, ObjectMapper mapper) {
-        super(mapper);
-        this.druidWebService = druidWebService;
+        this((i1, i2, i3) -> druidWebService, mapper);
```

Comment:
Maybe touch the @param druidWebservice  comment to say 'A single webservice for handling all requests' to draw distinction from the selector constructor format.